### PR TITLE
Enable Style/SpaceAroundOperators Rubocop

### PR DESCRIPTION
**Pull request by nikhilgupta1211**
#1256 
Added the changes to the rubocop.yml file.

### CODE REVIEW

**Comment by Ana06** [comment]
@nikhilgupta1211 you have to remove it from [.rubocop_todo.yml](https://github.com/openSUSE/osem/blob/master/.rubocop_todo.yml#L600) and then correct the offenses (all of them can be automatically correct) :wink:

File: .rubocop.yml [hunk 1]
```diff
@@ -102,6 +102,9 @@ Style/BracesAroundHashParameters:
   Enabled: true
 
 #
```
Comment (Ana06) [inline .rubocop.yml]: Could you add here a short comment saying what this cop is for?

Comment (nikhilgupta1211) [inline .rubocop.yml]: Sure

**Comment by Ana06** [comment]
@nikhilgupta1211 LGTM :+1:
Please squash all the commits into one (or at least the last one into the first one): https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/

**Comment by nikhilgupta1211** [comment]
done :)

**Pull request merged**